File: src/main/java/no/nav/syfo/narmesteleder/Naermesteleder.java
package no.nav.syfo.narmesteleder;

import lombok.Data;
import lombok.experimental.Accessors;
import no.nav.syfo.model.NaermesteLederStatus;

@Data
@Accessors(fluent = true)
public class Naermesteleder {
    public long naermesteLederId;
    public String naermesteLederAktoerId;
    public String orgnummer;
    public NaermesteLederStatus naermesteLederStatus;
    public String navn;
    public String mobil;
    public String epost;
}

File: src/main/kotlin/no/nav/syfo/narmesteleder/consumer/Naermesteleder.kt
package no.nav.syfo.narmesteleder.consumer

data class Naermesteleder(
    val naermesteLederId: Long?,
    val naermesteLederFnr: String,
    val orgnummer: String,
    val naermesteLederStatus: NaermesteLederStatus,
    val navn: String,
    val mobil: String?,
    val epost: String?
)
